src/components/Documentation/Articles.js

```diff
@@ -144,46 +173,41 @@ const Articles = ({
         </TopicExpander>
       )}
 
-      {Object.values(articles).map((article) =>
-        article.articles ? (
-          <ArticleList
-            key={`ArticleList${id}${article.id}`}
-            isCollapsed={isCollapsed}
-          >
-            <Articles
-              articles={article.articles}
-              initialTopicsState={initialTopicsState}
-              isNested
-              key={article.id}
-              title={article.title}
-              topicPath={article.topicPath}
-              topicState={topicState}
-              topicToggleHandler={topicToggleHandler}
-              activeItem={activeItem}
-              depth={depth + 1}
-            />
-          </ArticleList>
-        ) : (
-          <ArticleList
-            key={`ArticleList${id}${article.id}`}
-            isCollapsed={isCollapsed}
-          >
-            <Article
-              key={`Article${article.id}`}
-              isCollapsed={isCollapsed}
-              article={article}
-              depth={depth + 1}
-              activeItem={activeItem}
-            />
-          </ArticleList>
-        ),
-      )}
-    </li>
+      <ArticleList isCollapsed={isCollapsed}>
+        {Object.entries(articles)
+          .filter(([filename]) => filename !== "index")
+          // Only get second arg
+          .map(([, article]) =>
```

Comment:
I low key forgot you could do this. I usually do `([_, thing])`, which I prefer because it signifies that there's a value there that's not being used, but the eslint "no unused" doesn't have any exceptions.